model/work.py

```diff
@@ -1422,7 +1426,9 @@ def query_to_json_array(query):
                 LicensePool.id.label('licensepool_id'),
                 LicensePool.collection_id.label('collection_id'),
                 LicensePool.open_access.label('open_access'),
-                LicensePool.licenses_available.label('available') > 0,
+                (LicensePool.licenses_available > 0).label('available'),
+                (LicensePool.licenses_owned > 0).label('owned'),
+                work_quality_column,
```

Comment:
The 'main collection' facet uses a nested filter to apply a restriction that involves both LicensePool.open_access and Work.quality. The only way I could get this to work was to put a copy of Work.quality into each LicensePool sub-document. It looks like when you create a nested query on a subdocument, the main document is out of scope. I'm not 100% sure about this though.